crates/swift-bridge-ir/src/codegen/codegen_tests/vec_codegen_tests.rs

```diff
@@ -824,6 +824,210 @@ void* __swift_bridge__$Vec_SomeStruct$as_ptr(void* vec_ptr);
     }
 }
 
+mod transparent_struct_vec_support_without_derives {
```

Comment:
Minor request:

Mind adding `/// Verify that ...` doc comments to any codegen test modules that you add.

They can sometimes feel redundant but I find that they really help when trying to understand a test since it's easier to read and make sense of prose than long snake case identifiers.
Especially for new contributors that are trying to understand what's going on.

Reply:
Done [b08764e](https://github.com/chinedufn/swift-bridge/pull/193/commits/b08764eb93e9627ea61db3036e2f6e4a672e2072)